### PR TITLE
added simple call function to tokenizer, changed protgym script to wo…

### PR DESCRIPTION
**Pull request by Muedi**
…rk with said tokenizer

I also removed the whitespaces :)

### CODE REVIEW

**Comment by Muedi** [comment]
@jamaliki I have no idea how to ask for a reviewer and need to go, so I'll ping you here :D

**Comment by jamaliki** [comment]
Hey, this is great! Let me merge my new tokenizer branch, maybe we can add your call function to that one and then we can merge this branch? That way we won't have to go back and change this branch.

File: Protein-gym.py
Comment (jamaliki) [inline Protein-gym.py:19]: You can do `from protein_lm.tokenizer import EsmTokenizer, AptTokenizer` here :)

File: Protein-gym.py
Comment (jamaliki) [inline Protein-gym.py:120]: Once PR #14 get's merged, you don't have to define the tokens here, you can just do
```
tokenizer = AptTokenizer()
```

**Review by jamaliki — CHANGES_REQUESTED** [review]
Small nits, let's wait on PR #14

File: Protein-gym.py
Comment (Muedi) [reply Protein-gym.py]: Hi, I just pushed the changes :)

**Review by jamaliki — APPROVED** [review]
LGTM! Great work :)

**Comment by Muedi** [comment]
closing this one as I needed to rebase anyway :)

**Pull request closed without merging**
